### PR TITLE
[vcpkg] Add keyword Supports to Feature in CONTROL file and manifest

### PR DESCRIPTION
**Pull request by JackBoosY**
Continue to PR #8601, This PR will enable the pipeline to automatically test all features of all ports.

TODO list:
- [x] Add keyword `Supports`
- [x] Handle keywords Supports when parsing `Feature`
- [ ] Document
- [x] Test

### CODE REVIEW

**Comment by JackBoosY** [comment]
Note: Feature conflicts need to be considere.

**Comment by strega-nil** [comment]
Could you make certain to also add it to manifests? it shouldn't be too terrible, and if you want me to I can also push to this branch.

**Comment by JackBoosY** [comment]
@strega-nil Okay, but it seems that my changes didn't work. Could you please help me?

Thanks.

File: toolsrc/windows-bootstrap/vcpkg.props [hunk 1]
```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (strega-nil) [inline toolsrc/windows-bootstrap/vcpkg.props]: Why were the windows-bootstrap files deleted?

Comment (JackBoosY) [reply toolsrc/windows-bootstrap/vcpkg.props]: It looks like what you read is out of date. I've revert that.

**Review by strega-nil — APPROVED** [review]

**Review by BillyONeal — APPROVED** [review]

**Comment by JackBoosY** [comment]
@ras0219-msft @ras0219 Ping for review this PR.

**Comment by JackBoosY** [comment]
Close this PR temporary.

**Comment by JackBoosY** [comment]
Already added this feature.

**Pull request closed without merging**